src/sanitizer.js

```diff
@@ -109,11 +107,28 @@ const WHITELISTED_ATTRS = [
   'on',
   'placeholder',
   'option',
+  'submit-success',
+  'submit-error',
   /* Attributes added for amp-bind */
   // TODO(kmh287): Add more whitelisted attributes for bind?
   'text',
 ];
 
+/** @const {!Object<string, !Array<string>>} */
+const WHITELISTED_ATTRS_BY_TAGS = {
```

Comment:
Use a prototype-less object.

Reply:
Pushed to a later PR as we discussed offline